src/Keboola/StorageApi/Client.php

```diff
@@ -970,6 +970,15 @@ public function tableExists($tableId)
         }
     }
 
+    /**
+     * @param array $options
+     * @return array
+     */
+    public function searchTables($options = array())
+    {
+        return $this->apiGet("storage/search/tables?" . http_build_query($options));
```

Comment:
melo by to mit nejakou validaci, to tam nekdo napise `medatataKey` a bude se rozcilovat proc to nefunguje

Reply:
Udělám `SearchTablesOptions`